app/src/cli/run.rs

```diff
@@ -29,7 +29,7 @@ pub fn exec(cmd: Args) -> miette::Result<()> {
 }
 
 fn print_nf(nf: &ast::Exp) {
-    let mut stream = StandardStream::stdout(ColorChoice::Auto);
+    let mut stream = StandardStream::stdout(ColorChoice::Never);
```

Comment:
Consider setting `NO_COLOR` in the tests instead: 

* https://no-color.org/
* https://github.com/BurntSushi/termcolor?tab=readme-ov-file#automatic-color-selection

Reply:
Thanks, done in [d48c6e8](https://github.com/polarity-lang/polarity/pull/215/commits/d48c6e82cec14457af864a247bea8a506f010de8)